requirements.txt

```diff
@@ -29,4 +29,5 @@ werkzeug>=1.0,<=2.0.3
 multiprocess
 importlib-metadata==4.13.0
 sqlalchemy==1.4.37
-weasyprint==54.3
\ No newline at end of file
+weasyprint==54.3
+s3path
```

Comment:
Please lock the version of the new library to avoid unexpected breaks when the library updates in the future.